README.md

```diff
@@ -16,13 +16,22 @@ This "To-do" list allows you to mark prioity tasks listed by the user as complet
 
 - Github
 
-👤 👤 **TayyabKM**
+ 👤 **TayyabKM**
 
 - Github: [@TayyabKM](https://github.com/TayyabKM)
 - Twitter: [@TayyabI83126361](https://twitter.com/TayyabI83126361)
 - Linkedin: [@tayyab-irfan-01b72b1aa](https://www.linkedin.com/in/tayyab-irfan-01b72b1aa/)
 
 
+
+👤 **Wali Muhammad**
+
+- GitHub: [@](https://github.com/Itswali)
+- Twitter: [@](https://twitter.com/WaliMuh94818599)
+- LinkedIn: [@](https://www.linkedin.com/in/wali-muhammad-666040244/)
+
+
+
```

Comment:
- [ ] Good job so far team 👏 👏 your README looks good but there are some important sections that are missing. Microverse has provided us with the [UPDATED README TEMPLATE](https://github.com/microverseinc/readme-template) I would suggest that you use it and make sure that all the sections are covered 😃 
REMEMBER: 
 -- The only optional section is FAQ
 -- The Microverse logo in the  README template should be removed or replaced